mod1/src/file-2.xoo

```diff
@@ -0,0 +1,7 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// violate HasTag rule: xoo on line 7
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:41853/organizations/org3/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:41853/project/issues?id=sample&pullRequest=5887&issues=AWd6ErwDO_Y4hVnJ-p3r&open=AWd6ErwDO_Y4hVnJ-p3r)

[ISSUE_KEY]: AWd6ErwDO_Y4hVnJ-p3r